src/main/java/com/mentalhealthapi/mentalhealthapi/dao/RetrofitClientInstance.java

```diff
@@ -0,0 +1,20 @@
+package com.mentalhealthapi.mentalhealthapi.dao;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClientInstance {
+
+    private static Retrofit retrofit;
+    private static String BASE_URL = "";
+
+    public static Retrofit getRetrofitInstance() {
+        if (retrofit == null) {
+            retrofit = new retrofit2.Retrofit.Builder()
+                    .baseUrl(BASE_URL)
+                    .addConverterFactory(GsonConverterFactory.create())
+                    .build();
+        }
+        return retrofit;
+    }
```

Comment:
This is an example of a retrofit client instance that I believe will come in handy, as sourced from our lessons.